source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRViewController.h

```diff
@@ -0,0 +1,17 @@
+//
+//  ACRViewController.h
+//  ACRViewController
+//
+//  Created by jwoo on 5/26/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface ACRViewController : UIViewController
+
+@property NSString* jsonString;
+-(id) init: (NSString*) str;
```

Comment:
thanks

---
In reply to: [122552608](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122552608) [](ancestors = 122552608)